README.md

```diff
@@ -37,21 +45,22 @@ and recommend installing through [rustup](https://rustup.rs).
 - `cmake`. We recommend installing through homebrew, with `brew install cmake`.
```

Comment:
Actually, I'm not sure the onig crate requires cmake - I just checked `brew uninstall cmake` and it seemed to work. Anyone else want to verify?

Reply:
Looks like the `build.rs` of `onig_sys` stopped using cmake during summer.